utils/src/main/scala/za/co/absa/enceladus/utils/general/FileReader.scala

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright 2018-2019 ABSA Group Limited
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package za.co.absa.enceladus.utils.general
+
+import scala.io.Source
+
+trait FileReader {
+  def readFileAsLines(filename: String): Seq[String] = {
+    val sourceFile = Source.fromFile(filename)
+    try {
+      sourceFile.getLines().toList.map(x => x)
+    } finally {
+      sourceFile.close()
+    }
+  }
+
+  def readFileAsString(filename: String, lineSeparator: String = "\n"): String = {
```

Comment:
I feel like these methods belong to FsUtils object rather it being inside a trait. Is there a particular reason for a trait?

Reply:
Moving to `fs` folder (more appropriate) but keeping as separate `trait` as it's lightweight.

Reply:
Still Why `trait` and not an `object`? What are the benefits of requiring anyone who wants to use a function from FsUtils to mixin the trait?

Reply:
It crossed my mind originally then i dismissed it. 
Rethinking under the inspiration of the comment, it probably is a good adept for a static object instead of making it a class' "own" function.